packages/forklift-console-plugin/webpack.config.ts

```diff
@@ -76,6 +77,9 @@ export const ENVIRONMENT_DEFAULTS = {
    * Version of the plugin.  Defaults to the version in `package.json`.
    */
   VERSION: pluginMetadata.version,
+
+  TIMELINE_START: DateTime.now().minus({ days: 1 }).toISO(),
```

Comment:
I prefer not to do `TIMELINE_` things in the same place we do `PLUGIN_NAME` and `VERSION`, 
looking at where it's used, it has a fallback, can we drop the support for tweaking timeline from a ENV ?

Reply:
Added prefix `TEST_` to differentiate from production related env vars